zerver/lib/events.py

```diff
@@ -572,6 +572,9 @@ def fetch_initial_state_data(
             ).name
         )
 
+        if settings.ONBOARDING_VIDEO_URL is not None:
+            state["onboarding_video_url"] = settings.ONBOARDING_VIDEO_URL
```

Comment:
I think we want `None` values to be present here; we don't do optional keys in `register` much.

Reply:
Removed the `if settings.ONBOARDING_VIDEO_URL is not None:` conditional.

Is there any benefit to include such fields which don't have any value ? The response data will be slightly smaller in size (not much) if we just drop such fields.

Reply:
It can be simpler for clients to parse a response with a more predictable set of fields in the response.